x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/RoleMappingFileSettingsIT.java

```diff
@@ -163,23 +167,39 @@ public static void writeJSONFileWithoutVersionIncrement(String node, String json
         writeJSONFile(node, json, logger, versionCounter, false);
     }
 
-    private static void writeJSONFile(String node, String json, Logger logger, AtomicLong versionCounter, boolean incrementVersion)
+    public static void writeJSONFile(String node, String json, Logger logger, AtomicLong versionCounter, boolean incrementVersion)
```

Comment:
Nit: why did this become public?

Reply:
It's re-used by another test. However, I like your suggestion below to just move it to `ESIntegTestCase` which makes this bit obsolete.